### DISCUSSION TITLE
Not working with React Functional Component for me

### DISCUSSION THREAD

**Discussion by MohitAsswani** [gilbarbara/react-joyride#882] [General]
Here is my code

import Joyride from 'react-joyride';
...
...
return(
 <>
  <Joyride
        steps={steps}
      />
....
.......
<Button className='addClass'>Add More</Button>
</>
)

I also added debug as a prop and it is showing me all the data like props, steps and everything but no error. and also in the DOM it is rendered as -> <div class="react-joyride"></div>

**Comment by gilbarbara** [comment]
Hey @MohitAsswani 

Your example doesn't show Joyride usage.
Try to follow the Basic example from the [demo](https://github.com/gilbarbara/react-joyride-demo)

**Reply by MohitAsswani** [reply]
It is visible when I edit the same. Let me try to put it in this comment. Please let mw know if it is visible to you now.
Here you go:
import Joyride from 'react-joyride';
...
...
return(
 <>
  <Joyride
        steps={steps}
      />
....
.......
<Button className='addClass'>Add More</Button>
</>
)


I also added debug as a prop and it is showing me all the data like props, steps and everything but no error. and also in the DOM it is rendered as -> <div class="react-joyride"></div>

**Reply by MohitAsswani** [reply]
Otherwise Please try this link - https://docs.google.com/document/d/15iymnbypARqx9g-TrKxDSW1WGUsSA8Kl9XdEI3AmyZQ/edit?usp=sharing

**Comment by gilbarbara** [comment]
hey @MohitAsswani 
You need to use triple backticks to insert code in GitHub. Take a look at the [docs](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-code).

Anyway, it's impossible to know what is happening with your app without running it.
Try to create a minimal reproducible example at codesandbox.

**Reply by MohitAsswani** [reply]
Thanks @gilbarbara for your help. actually it was not working with conditional rendering for me but I somehow figured it out. Thanks again, for your help.
